### PR TITLE
update path url for dcp_pluto to bytes

### PR DESCRIPTION
**Pull request by mbh329**
Addresses issue 270 but limited to just `dcp_pluto`, One reviewer required 👍 

This PR addresses the url path change due to the server for Bytes of the Big Apple.

### CODE REVIEW

**Comment by mbh329** [comment]
To test, you can pull down the feature branch and test locally (without pushing to s3)

File: library/templates/dcp_pluto.yml
Comment (td928) [inline library/templates/dcp_pluto.yml:6]: extra space after the url line

**Review by td928 — CHANGES_REQUESTED** [review]
see comment above otherwise looks good

File: library/templates/dcp_pluto.yml
Comment (mbh329) [reply library/templates/dcp_pluto.yml:6]: nice catch

Comment (mbh329) [reply library/templates/dcp_pluto.yml]: @td928 should be good now

**Review by td928 — APPROVED** [review]
LGTM @mbh329 after reading Sasha's [manual on how to update the actions](https://github.com/NYCPlanning/action-library-archive/wiki/Overview-v1.1-and-v1.2). My understanding is we can test the image by making a staging image before even merging in the feature branch. We can do that together. Let me know how do you want to proceed with this.

**Pull request merged**
